js/packages/teams-ai/src/StreamingResponse.ts

```diff
@@ -53,42 +58,42 @@ export class StreamingResponse {
     }
 
     /**
-     * Sends an informative update to the client.
+     * Queues an informative update to be sent to the client.
      * @param {string} text Text of the update to send.
-     * @returns {Promise<void>} - A promise representing the async operation.
      */
-    public sendInformativeUpdate(text: string): Promise<void> {
+    public queueInformativeUpdate(text: string): void {
         if (this._ended) {
             throw new Error('The stream has already ended.');
         }
 
-        // Send typing activity
-        return this.sendActivity('typing', text, {
-            streamType: 'informative',
-            streamSequence: this._nextSequence++
-        });
+        // Queue a typing activity
+        this.queueActivity(() => ({
```

Comment:
wrt the informative update not being sent during yesterday's test session, should we move this into `queueNextChunk `instead